draft-westerlund-tsvwg-dtls-over-sctp-bis.md

```diff
@@ -388,11 +334,38 @@ TLS:  Transport Layer Security
    The new user_message', i.e the protected user message, is the input
    to SCTP.
 
-   On the receiving side DTLS is used to decrypt the records.  If a
-   DTLS decryption fails, the DTLS connection and the SCTP association
-   are terminated. Due to SCTP-AUTH preventing delivery of corrupt
-   fragments of the protected user message this should only occur in
-   case of implementation errors or internal hardware failures.
+   On the receiving side DTLS is used to decrypt the individual
+   records. There are three failure cases an implementation needs to
+   detect then act on as described below:
```

Comment:
Maybe `detect and then act on` is better?

Reply:
Yes. implemented.